src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -404,6 +404,18 @@ public function thisProductHasThisProductOption(ProductInterface $product, Optio
     }
 
     /**
+     * @Given there are :quantity items of product :product in variant :variant available in the inventory
+     */
+    public function thereAreItemsOfProductInVariantAvailableInTheInventory($quantity, ProductInterface $product, ProductVariantInterface $variant)
+    {
+        $indexOfVariant = $product->getVariants()->indexOf($variant);
+        $product->getVariants()->get($indexOfVariant)->setTracked(true);
+        $product->getVariants()->get($indexOfVariant)->setOnHand($quantity);
```

Comment:
We should use already existing transformer to obtain specific variant of a specific product. Then we can simply do:
```
$variant->setTracted(true);
$variant->setOnHand($quantity);
```